app/services/govuk_emails/email_monitor.rb

```diff
@@ -65,23 +68,21 @@ def send_email
       mailer.constantize.public_send(mail_method, *email_args).send(delivery_method)
     end
 
-    def capture_error
-      raise StandardError, error_message
-    rescue StandardError => e
-      Raven.capture_exception(e)
-    end
-
-    def error_message
-      [
-        '*Email ERROR*',
-        "*#{mailer}.#{mail_method}* could not be sent",
-        "*GovUk email status:* #{email.status}",
-        email_args.to_s
-      ].join("\n")
+    def send_undeliverable_alert(error)
+      failure_reason = if error.is_a?(StandardError)
+                         error.class.to_s
+                       else
+                         error
+                       end
+      UndeliverableEmailAlertMailer.notify_apply_team(email_address, failure_reason, @mailer, @mail_method, @email_args)
```

Comment:
It looks like we're removing the sentry alert and just relying on the email. I'm just wondering if it's worth keeping the sentry alert too. Personally I'm more likely to see the slack notification for that alert than the email (I'm not sure if I even have access to that account 🤔)

Reply:
Fair point.  That email is monitored by Lizzie and Alice, but it can't help us being notified too.  I'll make it a specialised error so that it stands out a bit more.